tools/confix/cmd/migrate.go

```diff
@@ -61,7 +61,12 @@ In case of any error in updating the file, no output is written.`,
 			targetVersion := args[0]
 			plan, ok := confix.Migrations[targetVersion]
 			if !ok {
-				return fmt.Errorf("unknown version %q, supported versions are: %q", targetVersion, slices.Collect(maps.Keys(confix.Migrations)))
```

Comment:
This is the correct way to specify it since Go 1.23. The alternative is more verbose but adds nothing.